packages/mui-docs/src/DocsProvider/DocsProvider.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { UserLanguageProvider } from '../i18n';
+
+export interface DocsConfig {
+  LANGUAGES: string[];
+  LANGUAGES_SSR: string[];
+  LANGUAGES_IN_PROGRESS: string[];
+  LANGUAGES_IGNORE_PAGES: (pathname: string) => boolean;
+}
+
+const DocsConfigContext = React.createContext<DocsConfig | null>(null);
+
+export interface DocsProviderProps {
+  config: DocsConfig;
+  defaultUserLanguage: string;
+  children?: React.ReactNode;
+}
+
+export function DocsProvider({ config, defaultUserLanguage, children }: DocsProviderProps) {
```

Comment:
Can we use either default or named exports consistently? Here, the DocsProvider is exported by name, while Link has a default one.
I prefer the named ones - there's no need to reexport both `*` and `default` from the index file, and it looks better when importing multiple identifiers from a module (intellisense gives all options in one place)

Reply:
You're right, didn't even think about it, just copied from the docs blindly 😄